hotel-api-persistence/src/main/java/com/fponce/hotelapp/persistence/impl/HotelRepositoryImpl.java

```diff
@@ -6,10 +6,9 @@
 import org.springframework.stereotype.Repository;
 
 import javax.sql.DataSource;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.sql.*;
```

Comment:
Avoid wildcards

Reply:
I think this could be solved checking the IntelliJ's configuration

Reply:
Done.